Resources/Prototypes/_DV/Augments/arm.yml

```diff
@@ -0,0 +1,238 @@
+  components:
+  - type: Item
+    storedRotation: 0
+  - type: Sprite
+    sprite: Objects/Misc/module.rsi
+    state: mainboard
+  - type: Tag
+    tags: [ AugmentPaperworkToolBelt ]
+
+- type: entity
+  parent: BaseItem
+  id: AugmentToolPanelCasing
+  name: tools panel augment casing
+  description: Can be implanted into a body to provide access to specific tools. This one is incomplete.
+  components:
+  - type: Sprite
+    sprite: _DV/Objects/Augments/Arm.rsi
+    state: base
+  - type: Item
+    size: Ginormous
+  - type: Construction
+    graph: CreateToolsPanel
+    node: casing
+  - type: Tag
+    tags: [ AugmentToolPanelCasing ]
+
+- type: entity
+  parent: BaseItem
+  id: AugmentToolPanelCasingCables
+  name: tools panel augment casing with cables
+  description: Can be implanted into a body to provide access to specific tools. This one is incomplete.
+  components:
+  - type: Sprite
+    sprite: _DV/Objects/Augments/Arm.rsi
+    state: base_cables
+  - type: Item
+    size: Ginormous
+  - type: Construction
+    graph: CreateToolsPanel
+    node: cables
+
+- type: entity
+  parent: AugmentBaseToolPanel
+  id: AugmentToolsPanel
+  name: tools panel augment
+  description: A panel that can be augmented into a patient's arm to provide them access to its contents. This one can hold tools.
+  components:
+  - type: Sprite
+    state: engineering
+  - type: Storage
+    whitelist:
+      tags:
+      - Wirecutter
+      - Crowbar
+      - Screwdriver
+      - Wrench
+      - GeigerCounter
+      - Powerdrill
+      - JawsOfLife
+      - HolofanProjector
+      - Multitool
+      - AppraisalTool
+      components:
+      - StationMap
+      - SprayPainter
+      - NetworkConfigurator
+      - RCD
+      - RCDAmmo
+      - Welder
+      - PowerCell
+      - Geiger
+      - TrayScanner
+      - GasAnalyzer
+  - type: Construction
+    graph: CreateToolsPanel
+    node: "utility tools panel"
+
+- type: entity
+  parent: AugmentBaseToolPanel
+  id: AugmentBotanyPanel
+  name: botanical tools panel augment
+  description: A panel that can be augmented into a patient's arm to provide them access to its contents. This one can hold botanical tools.
+  components:
+  - type: Sprite
+    state: botany
+  - type: Storage
+    whitelist:
+      tags:
+      - PlantSampleTaker
+      - BotanyShovel
+      - BotanyHoe
+      - BotanyHatchet
+      - Dropper
+      components:
+      - HandLabeler
+  - type: Construction
+    graph: CreateToolsPanel
+    node: "botany tools panel"
+
+- type: entity
+  parent: AugmentBaseToolPanel
+  id: AugmentPaperworkPanel
+  name: paperwork tools panel augment
+  description: A panel that can be augmented into a patient's arm to provide them access to its contents. This one can hold paperwork tools.
+  components:
+  - type: Sprite
+    state: paperwork
+  - type: Storage
+    whitelist:
+      tags:
+      - Folder
+      - HandLabeler
+      components:
+      - Stamp
+      - TapeRecorder
```

Comment:
add tape casettes too

Reply:
the items are immovable so you couldn't exactly insert a tape cassette into a recorder from an augment